File: qlty-cli/tests/lang/java/basic.in/cyclomatic/IterativeMap.java
import java.util.List;

class IterativeMap {
  public static void main(String[] args) {
    List<String> animals = List.of("dog", "cat", "bear");

    animals.stream().map(animal -> {
      System.out.println(animal);
      return animal;
    });
  }
}

File: qlty-cli/tests/lang/csharp/basic.in/cyclomatic/IterativeMap.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Cyclomatic
{
  class IterativeMap {
    public static void Main(string[] args) {
      List<string> animals = new List<string> { "dog", "cat", "bear" };

      animals.Select(animal => {
        Console.WriteLine(animal);
        return animal;
      }).ToList();
    }
  }
}